WcaOnRails/app/models/user.rb

```diff
@@ -155,6 +155,8 @@ def dummy_account?
     wca_id.present? && encrypted_password.blank? && email.casecmp("#{wca_id}@worldcubeassociation.org") == 0
   end
 
+  scope :delegate, -> { where.not(delegate_status: nil) }
```

Comment:
For the record, it seems like we're not actually consistent about pluralizing our scopes. For example, there's the scope [`not_dummy_account`](https://github.com/cubing/worldcubeassociation.org/blob/fa94cab57ffd9528092c1e9930cefb57d4b739e1/WcaOnRails/app/models/user.rb#L153) (singular) which returns all `Users` (plural) that are not dummy accounts.

http://stackoverflow.com/questions/18495000/singular-or-plural-scope-names-in-rails